### PR TITLE
Prac 6 new branch

### PR DESCRIPTION
**Pull request by ChristineYamin**
@DianaBukeikhanova
@Anastasia-Eremenko

### CODE REVIEW

File: prac_06/programming_language.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    def is_dynamic(self):
+
+        if self.typing == "Dynamic":
+            return True
+        else:
+            return False
+
+
+
```
Comment (DianaBukeikhanova) [inline prac_06/programming_language.py:28]: Good codes.

File: prac_06/used_cars.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    limo = Car("Limo",100)
+    limo.add_fuel(20)
+    print(f"The amount of fuel in the car {limo.fuel}")
+    limo.drive(115)
+    print(limo)
+
+
+
+
+
```
Comment (DianaBukeikhanova) [inline prac_06/used_cars.py:26]: Nice job!

File: prac_06/guitar.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    def __str__(self):
+        """Initialise a guitar instance."""
+        return f"{self.name} get_age() - Expected {self.get_age()}. Got {self.get_age()}\n{self.name} is_vintage() - Expected {self.is_vintage()}. Got {self.is_vintage()}"
+        # return f"{self.name} ,{self.year} : {self.cost}"
+
+
+    def get_age(self):
+        """ Calculate the age of guitar"""
+        current_year = 2024
+        return current_year-self.year
```
Comment (DianaBukeikhanova) [inline prac_06/guitar.py:22]: If you are adding fixed number, that will not change. It is better to use as a CONSTANT

File: prac_06/guitars.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+# guitar_cost = input(" Cost :")
+guitars.append(Guitar("Gibson L-5 CES", 1922, 16035.40))
+guitars.append(Guitar("Line 6 JTV-59", 2010, 1512.9))
+guitars.append(Guitar("Fender Stratocaster" ,2014 , 765.40 ))
+
+
+print("These are my guitars:")
+for i, guitar in enumerate(guitars, 1):
+    vintage_string = " (vintage)" if guitar.is_vintage() else ""
+    print(f"Guitar {i}: {guitar.name:>20} ({guitar.year}), worth ${guitar.cost:10,.2f}  {vintage_string}")
```
Comment (DianaBukeikhanova) [inline prac_06/guitars.py:22]: Good code

File: prac_06/guitars.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+Word Occurrences
+Estimate: 30 minutes
+Actual:   35 minutes
+"""
+
+from prac_06.guitar import Guitar
+
+guitars =[]
+print("My guitars!!")
+# guitar_name = input("Name :")
```
Comment (Anastasia-Eremenko) [inline prac_06/guitars.py:11]: good naming convention, clear and concise

File: prac_06/guitars.py [hunk 6]
```diff
@@ -0,0 +1,22 @@
+"""
+Word Occurrences
```
Comment (Anastasia-Eremenko) [inline prac_06/guitars.py:2]: Overall your code is correctly formatted, uses appropriate naming convention, lines are not excessively long and the functions follow SRP (with relevant docstrings).
 
So great job!

The only thing i notice is that the file title docstring is Word Occurrences, which confused me a bit

**Pull request merged**